modules/Users/language/en_us.lang.php

```diff
@@ -679,4 +679,12 @@
     'ERR_USER_FACTOR_SMTP_REQUIRED' => 'SMTP server settings required first.',
     'ERR_USER_FACTOR_CHANGE_DISABLED' => 'Current user is not able to change two factor authentication settings.',
     'LNK_IMPORT_CAMPAIGNS' => 'Import Campaign',
+    // for Google Token/Synchronization settings
+    'LBL_GOOGLE_API_SETTINGS' => 'Google Account Synchronization',
+    'LBL_GOOGLE_API_TOKEN' => 'Google API Token',
+    'LBL_GOOGLE_API_TOKEN_HELP' => 'Use this to authorize SuiteCRM to access your Google Calendar. This is required for synchronization. Don\'t forget to check the \'Enable Calendar Sync\' checkbox when done.',
```

Comment:
Instead of `\'Enable Calendar Sync\'` can't we have another sign to avoid escaping? as in `"Enable Calendar Sync"`. 
As a rule of thumb no code or characters that require escaping should be present. If you can't do that at all (as when using apostrophes) at least reduce the use of signs as markers

Language values should not require Escape sequences .
This to avoid bugs caused by translations that sometimes forget to escape things.
@Dillon-Brown Maybe placing this in SuiteCRM good practices?

Reply:
@horus68  @Dillon-Brown 
It is in the coding standards:
"Use single and double quotes when appropriate. If you’re not evaluating anything in the string, use single quotes. There should be no need to ever escape quotes within a string."
I'll fix it.

Reply:
@ApatheticCosmos an escaped single quote still existes in  `Don\'t forget to check`

Reply:
@horus68 Is using `&#39;` for a single quote acceptable here?